app/views/participants/_details.html.erb

```diff
@@ -20,6 +20,7 @@
 			<%= link_to "#{@participant.high_school.name} #{@participant.grad_year}", 
 					high_school_cohort_path(@participant.high_school, (@participant.grad_year || 0)), 
 					:class => 'redirect-link' unless @participant.high_school.nil? %>
+
```

Comment:
Could you remove this blank line and the extra whitespace on line 66, por favor? It's not a big issue, but lots of little places with extra whitespace can add up quickly.